stories/setup/config.ts

```diff
@@ -1,10 +1,15 @@
 import { configure } from '@storybook/react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fas } from '@fortawesome/free-solid-svg-icons';
 
 // Include application styles.
 import '../../src/styles.scss';
 // Apply some custom styles to storybook.
 import './styles.scss';
 
+// Import all the "free solid FontAwesome" icons
+library.add(fas);
```

Comment:
This has the potential to get out of sync with the real app so I'd rather see it moved to a shared function. I think all app configuration can be moved to a shared function. Storybook doesn't need all of this but it should be harmless to run the code:

```diff
diff --git a/src/index.tsx b/src/index.tsx
index caa7053..76faff4 100644
--- a/src/index.tsx
+++ b/src/index.tsx
@@ -11,22 +11,8 @@ import './styles.scss';
 import App from './components/App';
 import configureStore from './configureStore';
 
-if (process.env.NODE_ENV === 'production') {
-  // The second parameter prevents the log level to be persisted in a cookie or
-  // localStorage.
-  log.setLevel(log.levels.INFO, false);
-} else {
-  log.setLevel(log.levels.DEBUG, false);
-}
-
-if (process.env.REACT_APP_SENTRY_DSN) {
-  Raven.config(process.env.REACT_APP_SENTRY_DSN, {
-    logger: 'client',
-  }).install();
-}
-
-// Import all the "free solid FontAwesome" icons
-library.add(fas);
+configureApplication();
 
 const store = configureStore();
 
```

Reply:
ah thanks, I wanted to file an issue for that too or, at least, leave a comment about the ever-growing code in `src/index.tsx`, which isn't super testable...

Reply:
I will probably try to do that first, likely in a separate patch.

Reply:
Patch here: https://github.com/mozilla/addons-code-manager/pull/228